check-osp/osp_checks.py

```diff
@@ -107,22 +107,34 @@ def check_services_status(system, **kwargs):
     hosts_agents = dict()
     hosts_status = dict()
     services = kwargs['services']
-
     for host in hosts:
         # if a hostname contains localhost, we want to avoid trying to connect
         if 'localhost' in host:
             continue
-        ssh = ssh_client(host, username="root", password=system.password)
-        with ssh:
-            for service_name, status in services.iteritems():
-                service_status = is_service_in_status(ssh, service_name, status)
-                try:
-                    hosts_agents[host].update({service_name: service_status})
-                except KeyError:
-                    hosts_agents[host] = {service_name: service_status}
-
+        try:
+            service_for_host = services[host]
```

Comment:
I would add a check to make sure that the host in the yaml exists on the provider.

Reply:
Can do that. :+1:

Reply:
Upon further discussion offline, this was deemed not required.